website/docs/tutorials/deploying-java-app-with-database.md

```diff
@@ -0,0 +1,259 @@
+---
+title: Deploying a Java OpenLiberty application with PostgreSQL
+sidebar_position: 1
+---
+
+This tutorial illustrates deploying a [Java OpenLiberty](https://openliberty.io/) application with odo and linking it to an in-cluster PostgreSQL service in the minikube environment.
+
+There are two roles in this example:
+
+1. Cluster Admin - Prepare the cluster by installing the required operators on the cluster.
+2. Application Developer - Imports a Java application, creates a Database instance, and connect the application to the Database instance.
+
+## Cluster admin
+---
+
+This section assumes that you have installed [minikube and configured it](../getting-started/cluster-setup/kubernetes.md).
+
+We will be using Operators in this guide. An Operator helps in deploying the instances of a given service, for example PostgreSQL, MySQL, Redis.
+
+Furthermore, these Operators are "bind-able". Meaning, if they expose information necessary to connect to them, odo can help connect your component to their instances,
+
+See the [Operator installation guide](../getting-started/cluster-setup/kubernetes.md) to install and configure an Operator in the Kubernetes cluster, if you have not already done so.
```

Comment:
This part should go into guide on Operators in Architecture section. Can you add a TODO here so that we address it at some point?